docs/quickstart/20_setup_flowehr_locally.qmd

```diff
@@ -4,14 +4,26 @@ title: Set-up & deployment
 
 This guide will take you through the basic set up and deployment for FlowEHR in your environment.
 
-## Forking the repo
+## Get the repositories
+
+Repositories:
+1. [FlowEHR-Data-Pot](https://github.com/UCLH-Foundry/FlowEHR-Data-Pot) - fork or clone this repo. Data source deployments (Example data pipeline) - can use [Satellite](https://github.com/UCLH-Foundry/Satellite) for synthetic EMAP star schema
+2. [FlowEHR](https://github.com/UCLH-Foundry/FlowEHR) - this a template repo so you create your own repo from the template and customise it to your institution.
+3. [Powershell-Scripts](https://github.com/UCLH-Foundry/Powershell-Scripts) - allows VM Customisations of windows VMs
```

Comment:
might be worth having further down the list as something that might be useful

Reply:
looking at the [preview](https://github.com/UCLH-Foundry/Garden-Path/pull/35#issuecomment-1675083042), the list doesn't seem to be rendering as expected. I wonder if it needs to have a newline between the `Repositories:` and the ordered list?

<img width="1026" alt="image" src="https://github.com/UCLH-Foundry/Garden-Path/assets/8124189/0b02be30-5653-4e34-a75c-f1e338173ba9">